client/src/components/Profile.jsx

```diff
@@ -0,0 +1,150 @@
+import React, { Component } from 'react';
+import { Row, Col, Input, Button } from 'react-materialize';
+import { server_url } from '../config.json';
+
+
+export default class Profile extends Component {
+    constructor(props) {
+        super(props);
+
+        this.state = {
+            Edit: false,
+            msg: '',
+            user: [],
+        };
+
+        this.UpdateClick = this.UpdateClick.bind(this);
```

Comment:
You don't pass this methods to another components so you don't need to bind this (and call constructor too)